### ISSUE TITLE
How to skip the pre-training of static images and BK30L and go directly to VOS training? 

### ISSUE THREAD

**Issue by longmalongma** [hkchengrex/Mask-Propagation#12]
How to skip the pre-training of static images and BK30L and go directly to VOS training? What changes need to be made in the code and command line?

**Comment by hkchengrex** [comment]
Yes. Just start stage 2 without loading a pretrained model (--load_network).
